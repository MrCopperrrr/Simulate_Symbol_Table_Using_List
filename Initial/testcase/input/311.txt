BEGIN
INSERT f string
RPRINT
BEGIN
INSERT g number
RPRINT
INSERT h string
END
RPRINT
END
RPRINT