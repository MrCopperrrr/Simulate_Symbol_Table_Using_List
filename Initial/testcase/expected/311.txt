success
f//1
success
g//2 f//1
success
f//1
